scheduled-jobs/build/bug-sweeps/4.1/Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+properties(
+    [
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '',
+                numToKeepStr: '90'
+            )
+        ),
+	disableConcurrentBuilds(),
+    ]
+)
+
+description = ""
+failed = false
+
+b = build job: 'sweep', propagate: false, parameters: [
+    [
+	$class: 'StringParameterValue',
+	name: 'BUILD_VERSION',
+	value: '4.1'
+    ]
+]
+
+description += "${b.displayName} - ${b.result}\n"
+currentBuild.description = description.trim()
+currentBuild.displayName = "${b.displayName}"
+failed |= (b.result != "SUCCESS")
+currentBuild.result = failed?"FAILURE":"SUCCESS"
```

Comment:
This weirdness I just copied from another scheduled wrapper job (ose3.9 I think). It's not very pretty but I'm open to suggestions for improvements.

Reply:
I'm a bit partial to copying up the result from the build https://github.com/openshift/aos-cd-jobs/blob/master/scheduled-jobs/build/ose-master/Jenkinsfile#L10-L12
```
currentBuild.result = (b.result == "SUCCESS") ? "SUCCESS" : "FAILURE"
currentBuild.displayName = "[${b.result}] ${b.displayName}"
currentBuild.description = b.description
```
though it should probably have its own build id in there.

Reply:
OK.